Source/Core/VideoCommon/GXPipelineTypes.h

```diff
@@ -15,6 +16,12 @@ class NativeVertexFormat;
 
 namespace VideoCommon
 {
+// This version number must be incremented whenever any of the shader UID structures change.
+// As pipelines encompass both shader UIDs and render states, changes to either of these should
+// also increment the pipeline UID version. Incrementing the UID version will cause all UID
+// caches to be invalidated.
+constexpr u32 GX_PIPELINE_UID_VERSION = 1;
```

Comment:
Shouldn't this have a `// last changed in PR 1234` comment to cause merge conflicts when two branches both update to the same version?